models/model_constants.py

```diff
@@ -1,8 +1,8 @@
 # model Runner actions constants
 from DataProcessing import dataProcessingConstants
 TRACKING = 'tracking'
-RE_ID = 're-id'
-
-BASIC_ACTIONS = [TRACKING, RE_ID]
+RE_ID_EVAL = 're-id-eval'
+RE_ID_TRAIN = 're-id-train'
+BASIC_ACTIONS = [TRACKING, RE_ID_EVAL, RE_ID_TRAIN]
 DATA_PROCESSING_ACTIONS = [dataProcessingConstants.CLUSTER, dataProcessingConstants.TRACK_AND_CROP]
 CHOICES = BASIC_ACTIONS + DATA_PROCESSING_ACTIONS
```

Comment:
Add the re-id actions list to the CHOICES = BASIC_ACTIONS + DATA_PROCESSING_ACTIONS + RE_ID_ACTIONS ....
this will grow as we add more functionality  . 
But the main idea is still the same, constants for every submodule, concat with all other actions from other models.